### PR TITLE
Updated release notes for cab_package resource and provider.

### PR DESCRIPTION
**Pull request by Vasu1105**
Signed-off-by: Vasu1105 <vasundhara.jagdale@msystechnologies.com>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -30,6 +30,19 @@ Please see [https://docs.chef.io/release_notes.html](https://docs.chef.io/releas
...
+* Added cab_package resource and provider which supports the installation of CAB/cabinet files. Using this resource you can install and remove cab windows packages. Example:
+
+  ```ruby
+  cab_package 'Windows6.1-KB958488-x64' do
+    source 'C:\Users\xyz\AppData\Local\Temp\Windows6.1-KB958488-x64.cab'
+    action :install
+  end
+
+  cab_package 'Windows6.1-KB958488-x64' do
+    source 'C:\Users\xyz\AppData\Local\Temp\Windows6.1-KB958488-x64.cab'
```
Comment (btm) [inline RELEASE_NOTES.md]: can source be a URL or just a file? what happens if I put in a URL?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -30,6 +30,19 @@ Please see [https://docs.chef.io/release_notes.html](https://docs.chef.io/releas
 
 * Support for Solaris releases before 10u11 has been removed
 * Upgraded Ohai to 8.20 with new / enhanced plugins. See the [ohai changelog](https://github.com/chef-cookbooks/ohai/blob/master/CHANGELOG.md)
+* Added cab_package resource and provider which supports the installation of CAB/cabinet files. Using this resource you can install and remove cab windows packages. Example:
+
+  ```ruby
+  cab_package 'Windows6.1-KB958488-x64' do
```
Comment (Vasu1105) [inline RELEASE_NOTES.md]: yes

**Comment by btm** [comment]
@Vasu1105 can you update your sign-off line on the commit to include an email address? Right now it says"

> Signed-off-by: Vasu1105 <--remote>

File: RELEASE_NOTES.md (hunk 2 above)
Comment (btm) [inline RELEASE_NOTES.md]: you might change it in the example then to imply that, like
`cab_package 'Install .NET 3.5 sp1 via KB958488' do`

**Comment by Vasu1105** [comment]
Closing this PR and opened new one for same changes https://github.com/chef/chef/pull/5413 as there are some signed off related issues coming on this PR

**Pull request closed without merging**